.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -162,29 +162,29 @@ jobs:
           regex: '.*[0-9]+.[0-9]+.[0-9]+[-_]?rc[0-9]+'
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
         with:
           name: ${{ runner.os }}-wheels-${{ matrix.pl_backend }}-${{ matrix.arch }}.zip
           path: ./wheelhouse/*.whl
 
   upload-pypi:
     needs: linux-wheels-aarch64
-    matrix:
-        os: [ubuntu-latest]
-        arch: [aarch64]
-        pl_backend: ["lightning_qubit"]
-    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+          arch: [aarch64]
+          pl_backend: ["lightning_qubit"]
```

Comment:
Is there a need to mix 2 and 4 line indentation here?

Reply:
Locally they look the same to me. I will try to fix it.